day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+puts "Give me any number: "
+number = gets.chomp.to_i
+puts ""
+
+number.times do |i|
+  i += 1
```

Comment:
This accumulator is redundant I believe

Reply:
The reason I made that accumulator is that if I didn't it will always be short by 1. For example if I gave it a number of 20 the program will only go up to 19 and stop there. That's the reason I add 1 to every number that enters the block, for it can get to 20.